regtests/ww3_tp2.15/input_rho/ww3_shel.inp

```diff
@@ -0,0 +1,48 @@
+$ -------------------------------------------------------------------- $
```

Comment:
@ukmo-juan-castillo Could you add in the ww3_tp2.15/info about the added tests imput_rho, so later we can use it to check this feature. 
A related question, does this new regtest check the interpolation of the new fields?

Reply:
Added some information about the new test, which checks the interpolation of the new fields. Please let me know if it is clear.